docs/cmd/deploy.md

```diff
@@ -77,6 +77,10 @@ Following values are accepted, ordered from most verbose to least: `trace`, `deb
 
 It should be useful to enable more verbose logging when something doesn't work as expected, to better understand what's going on, and to provide more useful output logs when reporting containerlab issues, while making it more terse in production environments.
 
+#### filter
+
+The local `--filter` flag allows a user to specify a subset of nodes from the topology to deploy, instead of all (default). This can be useful e.g. in CI/CD test case scenarios, where resource constraints may prohibit the deployment of a full representative topology.
```

Comment:
we should also call the field in the cmd/<command>.go files all the same ... nodeFilter is my suggestion, since configFilter and deployFilter are not really clear what they do and it is tedious to match names for the same functionality between files.